CHANGELOG.md

```diff
@@ -29,7 +29,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 <!-- For changes in existing functionality.  -->
+<!-- TODO: Remove `/next/` from URL before release -->
 - Constellation operators are now deployed using Helm.
+- Updated the config version to v2. Check [how to migrate your config](https://constellation-docs.edgeless.systems/constellation/next/reference/config-migration).
```

Comment:
Should I create a ticket for this? :D

Reply:
Either a ticket as release blocker or change this URL already (and let the link checker fail until we release)

Reply:
AB#2657
Failing CI actions on main are not nice :)